extensions/wikia/WikiaMobile/WikiaMobileCategoryService.class.php

```diff
@@ -80,6 +80,7 @@ public function alphabeticalList() {
 			$title = $categoryPage->getTitle();
 			$category = Category::newFromTitle( $title );
 			$collections = $this->model->getCollection( $category, $format );
+			ksort( $collections['items'] );
```

Comment:
Nope. If I understand correctly what you want to achieve—e.g. array with keys `['P', 'T', 'Ś']` sorted to look like this:
```
[
  'P' => […],
  'Ś' => […],
  'T' => […]
]
```
I didn't find any easy way for such sort by keys.